torch/fx/experimental/proxy_tensor.py

```diff
@@ -590,6 +591,35 @@ def unwrap_proxy(self, e):
         else:
             return e
 
+    def create_node(self, *args, **kwargs):
+        '''
+        Add stack_trace metadata here if not already present,
+        by filtering out forward() stack frames
+        '''
+        node = super().create_node(*args, **kwargs)
+        if 'stack_trace' not in node.meta and node.op not in ["placeholder", "output"]:
+            user_frame = self._find_user_frame()
+            if user_frame:
+                summary = traceback.extract_stack(user_frame)
```

Comment:
I wonder if maybe we could try out [`CapturedTraceback`](https://github.com/pytorch/pytorch/pull/107358?fbclid=IwAR1u_DAexSNr1ASYLsRUJ0jS7xPiqSjSKlT2-txdkXRUPF-WD5HSaFXxYzw)? It seems to be a faster version of `traceback.extract_stack`